docs/plugins/ingest.asciidoc

```diff
@@ -101,9 +87,7 @@ Throws error when the field is not an array.
 --------------------------------------------------
 {
   "join": {
-    "fields": {
-      "joined_array_field": "other_array_field"
-    }
+    "joined_array_field": "-"
```

Comment:
@kimchy @martijnvg @clintongormley do you have any preferences surrounding this?

Whether these single field key:value configurations should move to a more explicit configurations with two fields, one representing the field, and one representing the value. In join's case, that value is the separator.